.github/workflows/node.js.yml

```diff
@@ -36,7 +33,7 @@ jobs:
     - uses: actions/setup-node@master
       with:
         node-version: '14'
-    - run: npm ci
+    - run: npm i
```

Comment:
I'm curious to know why revert to npm i instead of npm ci as npm i can upgrade the dependencies in the process.
If it's because of the lack of dev dependencies, it can be addressed by the --also-dev flag.

Reply:
Good catch. Will revert it.
Currently not on my computer, but it has todo with gh actions and a npm dependency of feathers-hooks-common from a github commit (instead of npm version tag). Github actions failed and I read that `npm i` could help. (see actions of https://github.com/fratzinger/feathers-utils/pull/11/commits/3ac146f835b9d5e0a92375b825f5c01d8513b5d1).
I reverted to current feathers-hooks-common version and added `//@ts-expect-error` tags to prevent that as of now.

Related to https://github.com/feathersjs-ecosystem/feathers-hooks-common/pull/623